src/base/projects/Blob.svelte

```diff
@@ -44,6 +49,11 @@
   $: if (line) {
     scrollIntoView(line);
   }
+
+  let content: MaybeHighlighted;
+  (async () => {
```

Comment:
If you don't like IIAFEs, you could do it the svelte way with `onMount`.

Reply:
1e545010a9ff419c4ecb5074f85b6e921604e78c